go-controller/pkg/ovn/policy.go

```diff
@@ -219,15 +219,20 @@ func (oc *Controller) updateStaleDefaultDenyACLNames(npType knet.PolicyType, gre
 				return err
 			}
 		}
+		var aclT aclType
+		if aclList[0].Direction == nbdb.ACLDirectionToLport {
+			aclT = lportIngress
+		} else {
+			aclT = lportEgressAfterLB
+		}
```

Comment:
I think I had this comment before, should we have a `aclTypeFromDirection` method?

Reply:
just making sure you decided not to act on this comment